tests/static/all_question_types.xml

```diff
@@ -31,6 +31,7 @@
 							<question27 />
 							<question28 />
 							<question29 />
+                            <question33 />
```

Comment:
Looks like this file uses tabs for indentation, but this line has spaces. Do you think it's worth updating that for consistency? (the same applies on other lines added below)